src/diffusers/modeling_flax_utils.py

```diff
@@ -183,6 +183,9 @@ def to_fp16(self, params: Union[Dict, FrozenDict], mask: Any = None):
         ```"""
         return self._cast_floating_to(params, jnp.float16, mask)
 
+    def init_weights(self, rng: jax.random.PRNGKey) -> Dict:
```

Comment:
I like that we don't allow the `input_shape` to be passed for now since it's much more restricted than Transformers, *i.e.* we should for now always be able to infer the correct shape from the config. This looks good to me!

Reply:
+1